server/api/lib/tags/light_rail.js

```diff
@@ -0,0 +1,32 @@
+const TAG_NAME = 'רכבת קלה';
+
+const doesTagApply = async (plan, tagsResources) => {
+    const getReturnDict = (stringInPlanName) => {
+      return {
+          plan_id: plan.id,
+          tag_id: tagsResources.tagNameToTagId[TAG_NAME],
+          display_score: 0, /* TODO: Add the correct display score here */
+          created_by_data_rules: JSON.stringify([{ rule: `has "${stringInPlanName}" in plan name` }])
+      };
+    };
+
+    const planName = plan.attributes.PL_NAME;
+
+    if (!planName.includes('צומת') && !planName.includes('ציר')) {
+
+        if (planName.includes('רכבת קלה')) {
+            return getReturnDict('רכבת קלה');
+        }
+        if (planName.includes('רק"ל')) {
+            return getReturnDict('רק"ל');
+        }
+
+    }
+
+    return null;
```

Comment:
No need to return null, it's the default, always returned

Reply:
I like to write it excplicitly. It's a part of the tags interface. If the tag doesn't apply, it should return null.
I don't want this property to be abstracted and forgotten.

After being said that, the default return of JS is "undefined". I wrote null because it means that "I decided not to return nothing", instead of "I didn't know what to return, so here's undefined stuff". 

I would like to keep the null as a part of the interface, but we can move to undefined and remove the return statement if you really want to do so.

Reply:
maybe return false? Up to you